collector/lib/HostInfo.h

```diff
@@ -146,6 +149,8 @@ struct KernelVersion {
   std::string release;
   // the entire version string (as in `uname -v`)
   std::string version;
+  // the kernel machine string (as in `uname -a`)
```

Comment:
[nit] `-a` gives everything, so a more precise command to get the machine would be `-m`